scripts/release.py

```diff
@@ -0,0 +1,199 @@
+import json
+import os
+import re
+import subprocess
+
+from dotenv import load_dotenv
+from github import Github
+
+
+"""This release notes script is built to create a release notes draft for release candidates, patches, and minor releases.
+
+Setup:
+1. Create a `.env` file in the scripts directory.
+2. Create Github token: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic
+3. Give the Github token repo, user, audit_log, and project permissions.
+2. Add `export GH_TOKEN=<github token>` to the `.env` file.
```

Comment:
We can add exporting our Github tokens into personal `~/.zshrc` config files, no need for creating a `.env` file in scripts. I would prefer setup requiring github tokens being exported one way or another, not necessarily requiring an `.env` file. What do you think?

Reply:
That's great, I just had issues doing that initially so switched to the .env file. Let me give the .zshrc way another shot.